compiler/noirc_frontend/src/tests.rs

```diff
@@ -3185,17 +3323,17 @@ fn trait_unconstrained_methods_typechecked_correctly() {
                 self
             }
 
-            unconstrained fn foo(self) -> u64;
+            unconstrained fn foo(self) -> Field;
         }
 
-        impl Foo for Field {
-            unconstrained fn foo(self) -> u64 {
-                self as u64
+        impl Foo for u64 {
+            unconstrained fn foo(self) -> Field {
+                self as Field
```

Comment:
Why was this test changed?

Reply:
Thanks for catching this: I had changed it to debug `check_cast` and need to change it back

Reply:
(Reverted)